l3kernel/l3text-case.dtx

```diff
@@ -692,40 +696,39 @@
     \@@_change_case_loop:nnnw {#2} {#3} {#4} #5
   }
 %    \end{macrocode}
-%   For titlecasing, we need to fully expand the new character to see if it
-%   is a letter (or active).
+%   For titlecasing, we need to fully expand the new codepoint to see if it
```

Comment:
Is it clear what "a codepoint" means here?  And how/when such a codepoint can be expanded? (Only functions and active characters are expandable.)

Reply:
Changed